quickstart/101-azurerm_dns_zone/providers.tf

```diff
@@ -0,0 +1,16 @@
+terraform {
+  required_version = ">=0.12"
+  required_providers {
+    azurerm = {
+      source  = "hashicorp/azurerm"
+      version = "~>2.0"
+    }
+    random = {
+      source  = "hashicorp/random"
+      version = "~>3.0"
+    }
+  }
+}
+provider "azurerm" {
+  features {}
```

Comment:
I don't think features is supported in 2.0.

Reply:
I don't recall exactly when the features block was added, but I ran the code and it works. Also, the features block is mentioned throughout the 2.x version page. Anyway, as you noted earlier, it should be v3+.

Reply:
Fixed.